### PR TITLE
Adding Week 13 Blogpost

### PR DESCRIPTION
**Pull request by ganimtron-10**
Adding weekly post

image

### CODE REVIEW

File: docs/source/posts/2022/2022-08-31-week-13-praneeth.rst [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+Also, the PR `#623 <https://github.com/fury-gl/fury/pull/623>`_ got merged as now the tests were passing after the update.
+
+Then there were a few changes that were required before any new feature was to be implemented.
+
+As we are now adding new features to the DrawPanel the current tests are becoming bigger and bigger.
+Due to this creating, debugging, and managing tests is becoming harder.
+So to keep things simple, separating the tests to validate individual parts and features of the DrawPanel. This was done in PR `#674 <https://github.com/fury-gl/fury/pull/674>`_.
+
+Along this, there was a redundant parameter called `in_progress` which was used to keep a check whether the shapes are added to the panel or not, but it was confusing, and discarding it didn't affect the codebase. So PR `#673 <https://github.com/fury-gl/fury/pull/673>`_ removed that parameter and separated the two functions.
```
Comment (m-agour) [inline docs/source/posts/2022/2022-08-31-week-13-praneeth.rst:27]: Hi @ganimtron-10,
Well written post. Only missing a comma before `which was used`

**Comment by skoudoro** [comment]
@xtanion ???

**Review by xtanion — COMMENTED** [review]
Hi @ganimtron-10, Nice blog. A few commas are missing.
PTAL at my comments below.

**Comment by skoudoro** [comment]
@ganimtron-10 ?

**Comment by ganimtron-10** [comment]
Hello @skoudoro ,
Sorry for the delay I have updated the posts!!

File: docs/source/posts/2022/2022-08-31-week-13-praneeth.rst [hunk 2]
```diff
@@ -0,0 +1,36 @@
+==========================================
+Week 13 - Seperating tests and fixing bugs
```
Comment (skoudoro) [inline docs/source/posts/2022/2022-08-31-week-13-praneeth.rst:2]: `Separating` instead

File: docs/source/posts/2022/2022-08-31-week-13-praneeth.rst [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+--------------------------
+This week I managed to fix the translation issue in PR `#653 <https://github.com/fury-gl/fury/pull/653>`_. This was happening because of the calculation error while repositioning the shapes. Now it works as intended.
+
+.. image:: https://user-images.githubusercontent.com/64432063/187058183-840df649-163e-44dd-8104-27d6c2db87a9.gif
+    :width: 400
+    :align: center
+
+Also, the PR `#623 <https://github.com/fury-gl/fury/pull/623>`_ got merged as now the tests were passing after the update.
+
+Then there were a few changes that were required before any new feature was to be implemented.
```
Comment (skoudoro) [inline docs/source/posts/2022/2022-08-31-week-13-praneeth.rst:21]: strange sentence, to reformulate

File: docs/source/posts/2022/2022-08-31-week-13-praneeth.rst [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+As we are now adding new features to the DrawPanel, the current tests are becoming bigger and bigger.
+Due to this creating, debugging, and managing tests is becoming harder.
```
Comment (skoudoro) [inline docs/source/posts/2022/2022-08-31-week-13-praneeth.rst:24]: `are` instead of `is`

**Review by skoudoro — COMMENTED** [review]
Thank you for the update @ganimtron-10,

See below the last comments

**Comment by skoudoro** [comment]
LGTM! Thank you @ganimtron-10, merging

**Pull request merged**